ftw/referencewidget/browser/search.py

```diff
@@ -45,15 +46,20 @@ def __call__(self):
         json_prep['batching'] = batch_html
 
         traversel_type = get_traversal_types(self.context)
-
+        plone = api.portal.get()
         for item in results:
             contenttype = 'contenttype-' \
                 + item.portal_type.replace('.', '-').lower()
 
             traversable = item.is_folderish and  \
                 (item.portal_type in traversel_type)
 
-            label = '{0} ({1})'.format(item.Title, item.getPath())
+            date = ''
+            if item.portal_type == 'ftw.news.News':
```

Comment:
@Nachtalb 
Please remove the dependency to ftw.news and check the brain for the `start` attribute.
Show content of the start attr if there is any value

Reply:
Do you mean like this? 
```python
date = ' (%s)' %  item.start if item.start else ''
```

Reply:
Or something similar, you may use  `getattr`

Reply:
> you may use `getattr`

Doesn't every brain have a index if you add a new one. The only difference would be it it has a value or not, or am I wrong? That would mean that the getattr doesn't give any advantage. And I can't just use a default value like '' for the start attribute, because I still have to add the `' (%s)'` __only__ if it has a value. 

Also I just realized I would still have to Localize it.
```Python
date = ' (%s)' % plone.toLocalizedTime(item.start) if item.start else '' 
```

Reply:
yep go ahead 😄